src/containers/b2b/RequisitionPageMain.tsx

```diff
@@ -441,31 +444,35 @@ class RequisitionPageMain extends Component<RouteComponentProps<RequisitionPageM
   }
 
   handleSelectAllItems() {
-    const { productsData } = this.state;
-
-    const { match } = this.props;
-    const listUri = match.params.uri;
-    const scope = localStorage.getItem(`${Config.cortexApi.scope}_oAuthScope`);
-
-    const { pages } = productsData.pagination;
-    const pagesArr = Array.from(new Array(pages), (val, index) => index + 1);
-
+    const { productsData, paginatorUri } = this.state;
+    const size = productsData.pagination.results;
     this.setState({ selectAllLoader: true });
-    const promises = pagesArr.map(pageIndex => cortexFetch(`/itemlists/${scope}/${listUri}/lineitems/pages/${pageIndex}?zoom=${elementZoomArray.sort().join()}`, {
-      headers: {
-        'Content-Type': 'application/json',
-        Authorization: localStorage.getItem(`${Config.cortexApi.scope}_oAuthToken`),
-      },
-    }).then(res => res.json()));
-    Promise.all(promises)
-      .then((res:any) => {
-        const products = res.filter(el => el._element).map(el => el._element).flat();
-        this.setState({ selectedProducts: products, selectAllLoader: false, showSelectAllPopup: false });
-      })
-      .catch((error) => {
-        // eslint-disable-next-line no-console
-        console.error(error.message);
-      });
+
+    login().then(() => {
+      cortexFetch(`${paginatorUri}?zoom=${elementZoomArray.sort().join()}&followlocation&format=standardlinks,zoom.nodatalinks`,
```

Comment:
Why are we hardcoding these params in the request? They're available as configs to be passed in through Cortex.js and appended to the URL optionally:
format=standardlinks,zoom.nodatalinks